src/encoded/static/scss/encoded/modules/_top_hits_search.scss

```diff
@@ -0,0 +1,155 @@
+.top-hits-search {
+    width: 100%;
+
+    @at-root #{&}__form {
+        position: relative;
+        display: flex;
+        margin: 3px 10px;
+        padding: 0;
+        border-radius: 14px;
+        background-color: #fff;
+        border: none;
+        box-shadow: none;
+
+        input[type="text"] {
+            display: block;
+            flex: 1 1 auto;
+            margin: 0;
+            padding: 5px 10px;
+            width: 220px;
+            border-radius: 0;
+            border: none;
+            background: transparent;
+            box-shadow: none;
+            transition: width 0.1s ease-out;
+            -ms-transition: width 0.1s ease-out;
+            -moz-transition: width 0.1s ease-out;
+            -webkit-transition: width 0.1s ease-out;
```

Comment:
AFAIK don’t really need the browser-specific props, but they don’t hurt either.

Reply:
Good to know, pulled from old search box definition.